src/components/common/PaymentButton.tsx

```diff
@@ -18,13 +18,13 @@ import { ProductType } from '../../types/product'
 import { AuthModalContext } from '../auth/AuthModal'
 import CoinCheckoutModal from '../checkout/CoinCheckoutModal'
 
-const StyleButton = styled(Button)<{ isMultiline?: boolean }>`
+const StyleButton = styled(Button)<{ line?: 'multiline' }>`
   span {
     display: none;
   }
 
   ${props =>
-    props.isMultiline &&
+    props.line === 'multiline' &&
```

Comment:
Don't include unrelated fixes in the same commit.

Reply:
OK, will be attention next time